services/configuration/src/api.rs

```diff
@@ -249,48 +234,46 @@ where
     }
 }
 
-impl Api for PublicConfigApi {
+impl Api for PublicApi {
     fn wire(&self, router: &mut Router) {
-
         let self_ = self.clone();
         let config_actual = move |_: &mut Request| -> IronResult<Response> {
-            let info = self_.get_actual_config();
+            let info = self_.actual_config();
```

Comment:
I don't like all these `let self_ = self.clone()`. What do you think about creating separate methods that takes `self` by value (similar to [this](https://github.com/exonum/exonum/blob/master/exonum/src/api/public/blockchain_explorer.rs#L126))?